### ISSUE TITLE
steamcmd_wrapper still broken

### ISSUE THREAD

**Issue by lucius-martius** [flathub/com.valvesoftware.Steam#1132]
I had the same issue in #1028 and just read that it was fixed in #1096, but I'm now getting a different error:

```
$ flatpak run --command=steamcmd com.valvesoftware.Steam +login foo bar
Traceback (most recent call last):
  File "/app/bin/steamcmd", line 8, in <module>
    sys.exit(main())
  File "/app/lib/python3.10/site-packages/steamcmd_wrapper.py", line 21, in main
    file_ignorer = FileIgnorer(STEAMCMD_BOOTSTRAP_DIR,
NameError: name 'FileIgnorer' is not defined. Did you mean: 'file_ignorer'?
```

OS: Gentoo Linux
Flatpak version: 1.14.4
Steam Version: 1.0.0.78

**Comment by rabidlogic** [comment]
I get a slightly different error, but same general broken. 

`
(deck@steamdeck site-packages)$ flatpak run --user --command=steamcmd com.valvesoftware.Steam//beta +login test
Traceback (most recent call last):
  File "/app/bin/steamcmd", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/app/lib/python3.11/site-packages/steamcmd_wrapper.py", line 24, in main
    shutil.copytree.copytree(
    ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'function' object has no attribute 'copytree'

`
OS: SteamOS
Flatpak Version: 1.12.4
Steam Version: 1.0.0.78-beta

**Comment by doraskayo** [comment]
Can you check again?

**Comment by rabidlogic** [comment]
Unfortunately, I'm getting the same error. 

This is on a completely fresh install.

**Comment by doraskayo** [comment]
@nanonyme, FYI

**Comment by rabidlogic** [comment]
@doraskayo I forgot to mention that I tried both beta and main. I could not get beta to install because the Platform dependency wasn't available. I'm still getting the same error on main.

**Comment by lucius-martius** [comment]
The `FileIgnorer` error seems gone now but I still get the `copytree` error @rabidlogic was getting.

**Comment by rabidlogic** [comment]
@lucius-martius I think I got this resolved in #1186 - Mind testing that build out?

**Comment by lucius-martius** [comment]
Looks good! For me it works without issue now. 👍
